srcpkgs/ardour/template

```diff
@@ -15,9 +15,9 @@ short_desc="Professional-grade digital audio workstation"
 maintainer="Andrea Brancaleoni <abc@pompel.me>"
 license="GPL-2.0-or-later"
 homepage="http://ardour.org"
-
 nocross="executes test binaries during configure"
+python_version=3
```

Comment:
Ardour installing a python script is a bug. We can remove this line and pick https://github.com/Ardour/ardour/commit/a5ec4fa3acd4837e5c5914d9ce67b7bc32d4fcfb instead.